front/app/containers/Admin/projects/components/ProjectRow/ProjectMoreActionsMenu.tsx

```diff
@@ -32,21 +33,26 @@ const ProjectMoreActionsMenu = ({
 }: Props) => {
   const { formatMessage } = useIntl();
   const { data: authUser } = useAuthUser();
+  const { data: project } = useProjectById(projectId);
 
   const { mutate: deleteProject } = useDeleteProject();
   const { mutate: copyProject } = useCopyProject();
 
   const [isCopying, setIsCopying] = useState(false);
   const [isDeleting, setIsDeleting] = useState(false);
 
-  if (isNilOrError(authUser)) {
+  if (!authUser || !project) {
     return null;
   }
 
-  const userCanDeleteProject = isAdmin(authUser);
+  const userCanDeleteProject =
+    isAdmin(authUser) || !project.data.attributes.first_published_at;
+
```

Comment:
Does this mean Folder Mods or Project Mods can't delete draft projects? :thinking:

Reply:
Admins can delete projects at any time. Folder and project moderators can only delete draft projects that have not been published yet. Otherwise they cannot delete them.